exporter/metric/README.md

```diff
@@ -13,26 +13,35 @@ Google Cloud Monitoring is a managed service provided by Google Cloud Platform.
 
 ## Usage
 
-Add `github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric` to the import list and set up `go.mod` file accordingly.
-
-```go
-import mexpoter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric"
-```
-
 After you import the metric exporter package, then register the exporter to the application, and start sending metrics. If you are running in a GCP environment, the exporter will automatically authenticate using the environment's service account. If not, you will need to follow the instructions in [Authentication](#Authentication).
 
 ```go
 
 import (
-    ... (omitted)
     "go.opentelemetry.io/otel/api/metric"
+    "go.opentelemetry.io/otel/label"
+    "go.opentelemetry.io/otel/sdk/metric/controller/push"
+    "go.opentelemetry.io/otel/sdk/resource"
+
+    mexporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric"
 )
 
 // Initialize exporter option.
-opts := []mexporter.Option{}
+projectID := os.Getenv("GOOGLE_CLOUD_PROJECT")
+opts := []mexporter.Option{
+    mexporter.WithProjectID(projectID),
+}
+popts:= []push.Option{
+    push.WithResource(
```

Comment:
Not sure if this is useful for the minimal example. Maybe we should fix the `InstallNewPipeline` to allow null popts instead? Alternatively, just make an empty slice.

Reply:
Yes, supporting `nil` is best ;-)

If I use `v0.11.0` then an empty slice generates an error:

```
2020/09/11 10:04:55 Error during exporting TimeSeries: currently the aggregator is not supported: &{{0 0} 123 [123]}
2020/09/11 10:04:55 rpc error: code = InvalidArgument desc = Request was missing field timeSeries.
```

If I use `v0.11.1-0.20200910093037-ef5e24b7fcfc` then an empty slice works. Will change to proactively permit.

Reply:
Fixed: 7e67ff1fca11b098d4a7410d207cbf18fd75ac37